feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -393,6 +393,18 @@ class BillingWrapper(
         }
     }
 
+    override fun getProperProductID(
+        productID: String,
+        purchaseToken: String,
+        amazonUserID: String?,
+        onSuccess: (properProductID: String) -> Unit,
+        onError: (PurchasesError) -> Unit
+    ) {
+        // This function actually does something in Amazon and gets the term SKU.
+        // In Google, there's no such concept
+        onSuccess(productID)
```

Comment:
could we make this the default implementation so BIllingWrapper doesn't have to override?

Reply:
Yes! Good idea